packs/data/pfs-season-2-bestiary.db/collapsing-ramp-9-10.json

```diff
@@ -0,0 +1,118 @@
+{
+    "_id": "XoE0tsQ4twaoNtBu",
+    "data": {
+        "attributes": {
+            "ac": {
+                "value": 0
+            },
+            "hardness": 0,
+            "hp": {
+                "details": "",
+                "max": 0,
+                "temp": 0,
+                "value": 0
+            },
+            "stealth": {
+                "details": "<p>(expert)</p>",
+                "value": 18
+            }
+        },
+        "creatureType": "",
+        "details": {
+            "description": "<p>Abyssal fire hits the stone beneath the ramp, causing a 15-foot-long section of it to collapse.</p>",
+            "disable": "<p>@Check[type:crafting|dc:31] (expert) to stabilize the masonry. Each attempt takes 2 actions.</p>",
+            "isComplex": false,
+            "level": {
+                "value": 8
+            },
+            "reset": "",
+            "routine": ""
+        },
+        "saves": {
+            "fortitude": {
+                "saveDetail": "",
+                "value": 0
+            },
+            "reflex": {
+                "saveDetail": "",
+                "value": 0
+            },
+            "will": {
+                "saveDetail": "",
+                "value": 0
+            }
+        },
+        "source": {
+            "value": "Pathfinder Society Scenario #2-22: Breaking the Storm: Excising Ruination"
+        },
+        "statusEffects": [],
+        "traits": {
+            "di": {
+                "custom": "",
+                "selected": {},
+                "value": []
+            },
+            "dr": [],
+            "dv": [],
+            "rarity": "uncommon",
+            "size": {
+                "value": "med"
+            },
+            "traits": {
+                "custom": "",
+                "value": [
+                    "environmental"
+                ]
+            }
+        }
+    },
+    "img": "systems/pf2e/icons/default-icons/hazard.svg",
+    "items": [
+        {
+            "_id": "fFkh0yEPdGL70sQI",
+            "data": {
+                "actionCategory": {
+                    "value": "interaction"
+                },
+                "actionType": {
+                    "value": "reaction"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p><strong>Trigger</strong> A creature walks onto the collapse site;</p>\n<p><strong>Effect</strong> The triggering creature falls off the Colossus and lands on the ground, taking [[/r {50}[bludgeoning]]]{50 bludgeoning damage}. They can use the @Compendium[pf2e.actionspf2e.Grab an Edge]{Grab an Edge} reaction to avoid falling (@Check[type:reflex|dc:24] save).</p>"
```

Comment:
```suggestion
                    "value": "<p><strong>Trigger</strong> A creature walks onto the collapse site;</p>\n<hr />\n<p><strong>Effect</strong> The triggering creature falls off the Colossus and lands on the ground, taking [[/r {50}[bludgeoning]]]{50 bludgeoning damage}. They can use the @Compendium[pf2e.actionspf2e.Grab an Edge]{Grab an Edge} reaction to avoid falling (@Check[type:reflex|dc:24] save).</p>"
```